### ISSUE TITLE
[web] Scrolling error in a11y mode in Desktop

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#45198]
Scrolling only works for going down on a list of contents. It does not work for going up.

Tested on: Safari, MacOS, VoiceOver

**Comment by nbayati** [comment]
Duplicate of #104036
